src/containers/AppPage/styles.js

```diff
@@ -47,18 +64,42 @@ export const useStyles = makeStyles((theme) => ({
     padding: theme.spacing(3),
   },
   circle1: {
-    height: 200,
-    width: 200,
+    position: "absolute",
+    left: -40,
+    bottom: 120,
+    height: 170,
+    width: 170,
     borderRadius: "50%",
-    backgroundColor: "#EF6A6A",
+    backgroundColor: "#FFAEC8",
+    zIndex: -100,
   },
   circle2: {
-    height: 200,
-    width: 200,
+    position: "absolute",
+    left: 30,
+    bottom: 10,
+    height: 170,
+    width: 170,
     borderRadius: "50%",
-    backgroundColor: "#EF6A6A",
+    backgroundColor: "#F9DAE4",
+    zIndex: -80,
+  },
+  logoutOpen: {
+    position: "absolute",
+    bottom: 65,
+    left: 40,
+    zIndex: -0,
+  },
+  logout: {
```

Comment:
great design intuition! I forgot to include the logout button in our UI drafts but happy with your placement and design